distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc

```diff
@@ -55,8 +57,31 @@ One way to accomplish this is to use the default <<org.codice.ddf.catalog.conten
 
 ===== High Availability Certificates
 
-Certificates need to have Subject Alternative Names (SANs) which will include the host for the HTTP proxy and for both ${branding} nodes.
-A certificate with SANs signed by the Demo CA can be obtained by following the steps at <<_creating_new_server_keystore_entry_with_the_certnew_scripts,Creating New Server Keystore Entry with the CertNew Scripts>>.
+Certificates need to have Subject Alternative Names (SANs) which will include the host for the failover
+proxy and for both ${branding} nodes. A certificate with SANs signed by the Demo CA can be obtained by
+navigating to `${home_directory}/etc/certs/` and, assuming the proxy's hostname is service.org, running
+the following for UNIX operating systems:
+
+[source]
+----
+./CertNew.sh -cn service.org -san "DNS:service.org"
+----
+
+or the following for Windows operating systems:
+
+[source]
+----
+CertNew -cn service.org -san "DNS:service.org"
+----
+
+[NOTE]
+====
+Systems that use DDF version 2.11.4 or later will automatically get a DNS SAN entry matching the CN
+without the need to specify the `-san` argument to the `CertNew` command.
+====
+
+More customization for certs can be achieved by following the steps at
+<<_creating_new_server_keystore_entry_with_the_certnew_scripts,Creating New Server Keystore Entry with the CertNew Scripts>>.
 This certificate is intended for testing purposes only, and should not be used in production.
```

Comment:
Can we maybe move this sentence up to the beginning of the self-signed certificates? To make it more prominent?

Reply:
Would a `[NOTE]` be better?

Reply:
👍  to both comments.